### ISSUE TITLE
deploy_cowrie.sh install script feed buglet

### ISSUE THREAD

**Issue by jebba** [threatstream/mhn#319]
tldr; I think you need to change from `database_hpfeeds` to this line in `deploy_cowrie.sh`:

`sed -i 's/#\[output_hpfeeds\]/[output_hpfeeds]/g' cowrie.cfg`

The `deploy_cowrie.sh` script runs `sed` on a few lines of `cowrie.cfg`. It fails to uncomment one line, which results in cowrie not feeding data back to the main mhn server. I think the cause is this line:

`sed -i 's/#\[database_hpfeeds\]/[database_hpfeeds]/g' cowrie.cfg`

The name of that section is now `[output_hpfeeds]`, so `sed` doesn't uncomment the line, which then leads to that section of the configuration not activating correctly, which means hits don't get fed to the main server.

**Comment by Antelox** [comment]
Can be closed: https://github.com/threatstream/mhn/pull/320
